bucket/stretchly.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.21.1",
+    "description": "Cross-platform open source app that reminds you to take breaks",
+    "##": "Not a portable app: settings are in $Env:AppData\\stretchly\\config.json",
+    "homepage": "https://github.com/hovancik/stretchly/",
```

Comment:
```suggestion
    "version": "0.21.1",
    "description": "Break time reminder",
    "homepage": "https://hovancik.net/stretchly/",
```

Reply:
I force-pushed this suggestion to my branch.